controller/api/models.py

```diff
@@ -517,12 +521,24 @@ def _default_scale(self, user, release):
         self.scale(user, structure)
 
     def logs(self, log_lines=str(settings.LOG_LINES)):
-        """Return aggregated log data for this application."""
```

Comment:
Need to keep this or write a new docstring, according to [code standards](http://docs.deis.io/en/latest/contributing/standards/#code-standards):
>  Docstrings and tests are required for all public methods, although the `flake8` tool used by Deis does not enforce this.